### DISCUSSION TITLE
Togglable shelf for downloads

### DISCUSSION THREAD

**Discussion by TCMine** [Floorp-Projects/Floorp#1269] [Feature Request]
I know there are extensions that attempt to replicate the old Chromium download shelf, but they all fall short in both looks and function. The sidebar download menu is also just too much. It would be nice if there was a native option (maybe as a status bar item) that could do all that the Chromium one did.

**Comment by RatWasHere** [comment]
im so up for this

please add

**Comment by TheMonDon** [comment]
Please Add, Thanks.

Love, Dad.

**Comment by creeper-0910** [comment]
Which UI are you talking about?

**Reply by TCMine** [reply]
![image](https://github.com/user-attachments/assets/f4e3d773-f99c-430e-8728-5694e2ba1611) 
![image](https://github.com/user-attachments/assets/1d97069c-bc06-463e-bfa6-c7b7ae5a9df3)

This is where downloads used to display in Chrome, on the bottom of the screen.

**Reply by creeper-0910** [reply]
about:config
→floorp.browser.native.downloadbar.enabled

**Reply by TCMine** [reply]
I did not know this existed at all, thank you.
